capirca/lib/paloaltofw.py

```diff
@@ -238,8 +238,6 @@ def pan_ports(ports):
       saddr_check = sorted(saddr_check)
       for addr in saddr_check:
         options["source"].append(str(addr))
-    else:
```

Comment:
Since we usually handle most address processing at once, but in this case we're not, it'd be helpful to leave a comment here (and for Destination-address processing) that the "any" case is handled later.